docs/reach/rs2/quickstart/base-rover-setup.md

```diff
@@ -1,133 +1,133 @@
+Agora vamos configurar o rádio LoRa na unidade rover para receber correções. 
 
 * Go to **Correction input** tab
 
-* In the **Base correction** pick **LoRa**
+* Em **Base correction** escolha **LoRa**
 
-* Frequency and air rate settings must match what was configured on the base
+* As configurações de frequência e air rate devem corresponder ao que foi configurado na base
 
-* Apply changes and you will see rover is connected to the base
+* Aplique as alterações e você verá que o rover está conectado à base
 
 <div style="text-align: center;"><img src="../img/quickstart/base-rover-setup/reachrs2-rover-lora.png" style="width: 800px;"></div>
 
 !!! tip ""
-    To make sure that corrections are passing from base to rover, you can put both receivers by the window for a few minutes to provide the sky visibility. Go to the status tab on the rover unit. Colorful bars are standing for available satellites. If LoRa is configured correctly, they will be accompanied by grey bars. These are standing for the corrections received from the base station.
+    Para garantir que as correções estejam passando da base para o rover, você pode colocar os dois receptores pela janela por alguns minutos para fornecer visibilidade ao céu. Vá para a guia Status na unidade rover. Barras coloridas estão disponíveis para os satélites disponíveis. Se o LoRa estiver configurado corretamente, eles serão acompanhados por barras cinzas. Eles representam as correções recebidas da base.
 
 
-### Placing Reach RS2 module
+### Implantação do módulo Reach RS2
 
-Once the units are configured, we can move to the field. For the field works, you will need a tripod and survey pole.
+Uma vez configuradas as unidades, podemos ir para o campo. Para os trabalhos de campo, você precisará de um tripé e um bastão.
 
-Mount Reach RS2 base and accurately level the tripod. Put the rover on the pole and attach the LoRa antennas. Turn on the devices.
+Monte o Reach RS2 da base e nivele com precisão o tripé. Coloque o rover no bastão e conecte as antenas LoRa. Ligue os dispositivos.
 
-!!! danger "Attention"
-    There **should be no** obstacles near the antenna that could block the sky view higher than 30 degrees above the horizon.
-    **Do not** test the device indoors or near buildings, do not cover the sky view for the antennas with laptops, cars or yourself. RTK requires good satellite visibility and reception.
+!!! danger "Atenção"
+    Não deve haver obstáculos perto da antena que possam bloquear a visão do céu a mais de 30 graus acima do horizonte.
+    **Não** teste o dispositivo em ambientes fechados ou próximos a edifícios, não cubra a vista do céu para as antenas com laptops, carros ou você mesmo. O RTK requer boa visibilidade e recepção de satélite.
 
-A guide on how to properly place the antennas is available in [Reach RS2 placement](../placement.md) section.
+Um guia sobre como posicionar corretamente as antenas está disponível na seção [Posicionando o Reach RS2](../placement.md).
```

Comment:
Um guia sobre como posicionar corretamente as antenas está disponível na seção [Montando o Reach RS2](../placement.md).

Reply:
This part wasn't added to the translation. Please close this PR and open another one